### ISSUE TITLE
Helm2: Upgrade golang to 1.13

### ISSUE THREAD

**Issue by karuppiah7890** [helm/helm#6379]
Output of `helm version`:
```
Client: &version.Version{SemVer:"v2.14.3", GitCommit:"0e7f3b6637f7af8fcfddb3d2941fcc7cbebb0085", GitTreeState:"clean"}
```

There's a PR for v1.12.9 #6334, hopefully we can change that to v1.13 by asking @ialidzhikov

**Comment by karuppiah7890** [comment]
The new PR has golang v1.13 😄

**Comment by karuppiah7890** [comment]
This can be closed as #6577 is merged :)
